.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "trailingComma": "none",
+  "tabWidth": 2,
+  "semi": false,
```

Comment:
No semicolons??

...I'll bend to the will of the people.

Reply:
It makes me sad too. Just wanted it to be consistent with the Brave Core one.

@petemill?

Reply:
We already have no semicolons in this repo and most(?) of Brave JS repos - blame StandardJS!